spec/decorators/plant_population_submission_decorator_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe PlantPopulationSubmissionDecorator do
+  let(:sd) do
+    PlantPopulationSubmissionDecorator.decorate(
+      create(:submission, submission_type: :population)
+    )
+  end
+
+  describe '#label' do
+    it 'does not throw exception' do
+      expect { sd.label }.
+        not_to raise_error
+    end
+
+    it 'returns proper label' do
+      pending 'Cannot test full label without solving #12'
+      fail
+    end
+
+    it 'returns population name when no species' do
+      sd.object.content.update(:step01, name: 'pn')
+      expect(sd.label).to eq '<span class="title">pn</span>'
+    end
+
+    it 'returns empty label when no name or species given' do
+      expect(sd.label).
+        to eq '<span class="title">' +
+              I18n.t('submission.empty_label') +
+              '</span>'
+    end
+  end
+
+  describe '#further_details' do
+    it 'does not throw exception' do
```

Comment:
Does it add any value? In other test cases you call this method and if anything is thrown these cases will fail.